tests/unit/type/stream.tcl

```diff
@@ -525,6 +534,15 @@ start_server {
         }
         assert_error ERR* {r XTRIM mystream MAXLEN 1 LIMIT 30}
     }
+
+    test {XTRIM with LIMIT delete entries no more than limit} {
+        r del mystream
+        r config set stream-node-max-entries 2
+        for {set j 0} {$j < 3} {incr j} {
+            r XADD mystream * xitem v
+        }
+        assert {[r XTRIM mystream MAXLEN ~ 0 LIMIT 1] == 0}
```

Comment:
@huangzhw shouldn't XTRIM return (trim) 1 in that case? because the second lp has only one entry which is exactly the LIMIT